assets/aistudio/environments/ai_studio_dev/context/requirements.txt

```diff
@@ -11,3 +11,6 @@ langchain-openai
 semantic-kernel
 pytest
 keyring
+# dev dependencies
+setuptools>70.0.0
```

Comment:
For some reasons, if we don't pin min version, it'll default to 69.x.x, which cause security issue